### ISSUE TITLE
Image descriptions are not copied

### ISSUE THREAD

**Issue by schuelermine** [zedeus/nitter#587]
Users on Twitter can choose to add an image description (alt text) to their images in tweets. This text is important for persons with impaired vision. Although standard HTML has a dedicated way to expose this text, Nitter does not currently transclude the description. In fact, the alt text is explicitly set to the empty string, which causes many screen reader programs to completely forgo describing the presence of an image. I think this should be changed, if it’s possible to retrieve the alt text.

**Comment by tux93** [comment]
I think this is a duplicate of #559
